style.css

```diff
@@ -24,46 +26,176 @@ body {
   font-size: 18px;
   color: #6070ff;
-  margin: 3%;
-  margin-top: 2.5%;
-  width: 76%;
-  height: 50px;
   cursor: pointer;
-  float: left;
-  transition: 0.7s;
+  margin-left: 12px;
 }
 
-.logo:hover {
-  font-size: 19px;
-  transform: translate(0, -1px);
+.menu-list {
+  margin-left: -200%;
+  transition: all 0.3s ease-in-out;
+  background-color: #fff;
 }
 
-.nav {
-  flex: 1;
-  text-align: right;
-  margin-right: 12px;
+ul {
+  display: flex;
+  list-style-type: none;
 }
 
-.menu-list {
+nav ul {
   display: none;
 }
 
-.menu {
+.skills a {
+  text-decoration: none;
+}
+
+.skills .extra {
+  text-decoration: none;
+  display: flex;
+  flex-direction: column;
+  align-content: center;
+}
+
+.extra a {
+  background: #e5e5e5;
+  text-decoration: none;
+  display: flex;
+  flex-direction: row;
+  flex-wrap: nowrap;
+  align-content: center;
+  gap: 2%;
+  height: 65px;
+  margin-right: 12px;
+}
+
+.menu-list.active {
+  scale: 100%;
+  display: flex;
+  flex-direction: column;
+  position: absolute;
+  margin-left: -0.1px;
+  top: -0.1px;
+  width: 100%;
+  height: 100%;
+  transition: all 0.3s ease-in-out;
+  background: rgba(47, 47, 197, 0.8);
+}
+
+.menu-list.active ul {
+  display: flex;
+  flex-direction: column;
+  gap: 10px;
+  margin: 4%;
+  margin-top: 40px;
+}
+
+.grid-item-section .category li a {
+  color: #6070ff;
+  text-decoration: none;
+}
+
+.menu-list.active ul li a {
+  text-decoration: none;
+  color: #fff;
+  font-size: x-large;
+  text-align: left;
+}
+
+li:hover {
+  transform: translate(0, -1px);
+  text-decoration: underline;
+  transition: 1s;
   cursor: pointer;
-  transition: ease-in;
 }
 
-.menu:hover {
+.grid-item-section .sub-title {
+  display: flex;
+  flex-wrap: nowrap;
+  height: 16px;
+  width: 100%;
+  gap: 1%;
+  font-weight: 400;
+  font-size: 12px;
+  margin-bottom: 4%;
+}
+
+.grid-item-section .sub-title li + li::before {
+  content: url("image/Counter.png");
+  margin: 8px;
+}
+
+.grid-item-section .sub-title li {
+  color: #486493;
+  background: #fff;
+  font-weight: 700;
+  font-size: 14px;
+}
+
+.grid-item-section .category li {
+  text-decoration: none;
+  background: #e6e7f1;
+  color: #6070ff;
+  border-radius: 7px;
+  font-weight: 400;
+  font-size: 13px;
+  padding: 4px 9px;
+  justify-content: center;
+  align-content: center;
+}
+
+.grid-item-section .category li:hover {
+  background: #6070ff;
   transform: translate(0, -1px);
+  transition: 0.5s;
+  cursor: pointer;
+}
+
+.grid-item-section .category li a:hover {
+  color: #fff;
+}
+
+.navigation {
+  display: block;
+}
+
+.bar {
+  display: block;
+  width: 30px;
+  height: 2px;
+  margin: 5px auto;
+  -webkit-transition: all 0.3s ease-in-out;
+  transition: all 0.3s ease-in-out;
+  background-color: #6070ff;
+  cursor: pointer;
+}
+
+.navigation.active .bar {
+  background-color: #fff;
+}
+
+.navigation.active .bar:nth-child(2) {
+  opacity: 0;
+}
+
+.navigation.active .bar:nth-child(1) {
+  transform: translateY(9px) rotate(45deg);
+}
```

Comment:
- [ ]  You should use font weight, size, and color from The provided Figma design.  Your design looks nice, but you miss something small to resemble it the one from Figma design


Your design            |  Figma design
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/43877689/174787881-a9d03cc4-72bf-44d3-ae00-b479f92388b2.png)  |  ![image](https://user-images.githubusercontent.com/43877689/174787999-5bc530b6-79e1-491a-b99d-dc0b0e5a90cc.png)

Reply:
@ishimwezachee, I thank you for reviewing my project and I also thank you for your motivating feedback.